lib/service/canvas_client_service_v2.dart

```diff
@@ -75,6 +76,7 @@ class CanvasClientServiceV2 {
       await _db.save(device, device.deviceId);
       unawaited(connectToDevice(device));
       log.info('CanvasClientService: Added device to db ${device.name}');
+      injector<CanvasDeviceBloc>().add(CanvasDeviceGetDevicesEvent());
```

Comment:
this event is expensive, 
if we add here, we should avoid calling this some places else

Reply:
we just call when open from castButton.